src/AzureAccount.api.d.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Copied from vscode-azure-account/src/azure-account.api.d.ts
```

Comment:
Are we not able to get their extension (or the typings from it) without making a copy?

I think even if we have a link to their repo/npm package somewhere tsc can figure out that we're only using the types and won't add their source to our output afaik.

Reply:
From [their sample](https://github.com/microsoft/vscode-azure-account/blob/master/sample/src/extension.ts#L2), it doesn't seem like that's possible:

> `import { AzureAccount, AzureSession } from '../../src/azure-account.api'; // Other extensions need to copy this .d.ts to their repository.`